core_impl/src/test/java/com/google/instrumentation/stats/StatsManagerImplTest.java

```diff
@@ -63,10 +66,24 @@ public void testGetNonexistentView() throws Exception {
     statsManager.getView(RpcConstants.RPC_CLIENT_REQUEST_COUNT_VIEW);
   }
 
-  // TODO(sebright): Turn this into a real test once StatsManagerImpl.record(...) is implemented.
-  // It currently only shows that record(...) can be called with a SimpleEventQueue.
   @Test
   public void testRecord() {
+    statsManager.registerView(RpcConstants.RPC_CLIENT_ROUNDTRIP_LATENCY_VIEW);
+    statsManager.record(
+        StatsContextFactoryImpl.DEFAULT,
```

Comment:
nit this method and the DEFAULT parameter kindof distracting. it seems it is only giving tags, which makes a lot more sense than guessing what `StatsContextFactoryImpl.DEFAULT` is all about.

can this be refactored? It would be nice to have this whole method intuitive, and this part is not